code/game/verbs/who.dm

```diff
@@ -7,63 +7,68 @@
 	var/list/Lines = list()
 
 	//for admins
-	var/living = 0
-	var/ghosts = 0
-	var/lobby = 0
-	var/livingAntags = 0
+	var/living = 0 //Currently alive and in the round (possibly unconscious, but not officially dead)
+	var/dead = 0 //Have been in the round but are now deceased
+	var/observers = 0 //Have never been in the round (thus observing)
+	var/lobby = 0 //Are currently in the lobby
+	var/living_antags = 0 //Are antagonists, and currently alive
+	var/dead_antags = 0 //Are antagonists, and have finally met their match
 
-	if (holder)
-		for (var/client/C in clients)
+	if(holder)
```

Comment:
Stop making formatting changes for reasons other than repo standards. You have no reason to make these edits and we seem to be getting into edit wars over files. It's not relevant to the changes you made.